src/sql/strong-oracle/create.sql

```diff
@@ -0,0 +1,332 @@
+BEGIN
```

Comment:
Generally you want to test if the object exists first and then drop else move on 
not attempt to drop then raise error if it bombs.
```
declare
    table_count NUMBER :=0;  
BEGIN
    select count(*) into table_count from all_tables where table_name='YOUR_TABLE_NAME' ;
    if table_count >= 1 then
       EXECUTE IMMEDIATE 'DROP TABLE YOUR_TABLE_NAME      cascade constraints'; 
    end if;
end;
/

```